### PR TITLE
Adds documentation for kubernetes daemon sets

### PR DESCRIPTION
**Pull request by m-orfanos**
#### Description

Adds documentation for kubernetes daemon sets

#### Related PRs

https://github.com/cloudops/cloudmc-kubernetes-plugin/pull/26
https://github.com/cloudops/cloudmc-kubernetes-sdk/pull/53

### CODE REVIEW

**Comment by Shabirmean** [comment]
@m-orfanos - Meant to add us as reviewers?

File: source/includes/gcp/_k8_daemonsets.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+### Daemon Sets
+
+<!-------------------- LIST DAEMON SETS -------------------->
+
+#### List daemon sets
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/k8s/anenvironment/daemonsets"
```
Comment (Shabirmean) [inline source/includes/gcp/_k8_daemonsets.md]: `anenvironment` ---> `an_environment`: I guess without the underscore at first site its not evident its an env

File: source/includes/gcp/_k8_daemonsets.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+```
+
+> The above command returns JSON structured like this:
```
Comment (Shabirmean) [inline source/includes/gcp/_k8_daemonsets.md]: `returns JSON ` ---> returns **a** JSON

File: source/includes/gcp/_k8_daemonsets.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+Note that the list is not complete, since it is refering to the [kubernetes api details](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md).
+
+<!-------------------- GET A DAEMON SET -------------------->
+
+#### Get a daemon set
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/k8s/anenvironment/statefulsets/test-aerospike/auth"
```
Comment (Shabirmean) [inline source/includes/gcp/_k8_daemonsets.md]: Same comment as above

File: source/includes/gcp/_k8_daemonsets.md (hunk 1 above)
Comment (m-orfanos) [reply source/includes/gcp/_k8_daemonsets.md]: a4d0afc081c7fa8eacad5f473c9865970cb197d8

File: source/includes/gcp/_k8_daemonsets.md (hunk 3 above)
Comment (m-orfanos) [reply source/includes/gcp/_k8_daemonsets.md]: a4d0afc081c7fa8eacad5f473c9865970cb197d8

File: source/includes/gcp/_k8_daemonsets.md (hunk 2 above)
Comment (m-orfanos) [reply source/includes/gcp/_k8_daemonsets.md]: 9c326f98862cb7914a6a02646fd99a94fc9b3c9a

**Review by y0geshdev — APPROVED** [review]
LGTM

**Review by Shabirmean — APPROVED** [review]
LGTM! Thank you for the changes 💯

**Pull request merged**